package.json

```diff
@@ -37,21 +35,36 @@
     "checkbundlesize": "npm run build:prod:standalone:minified && bundlesize",
     "mockserver": "mockserver -serverPort 1080 -jvmOptions \"-Dmockserver.enableCORSForAllResponses=true\""
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "echo 'Running pre-commit scripts...' && lint-staged",
+      "pre-push": "echo 'Running pre-push scripts...' && npm run test"
+    }
+  },
+  "lint-staged": {
+    "./*.js": [
+      "eslint --fix",
+      "git add"
```

Comment:
Is this going to add files that aren't in the original commit?

Reply:
If I understand what you mean by "original commit", then no. With lint-staged, what we're basically saying is: of the staged files that are attempting to be committed, find the files that match `./*.js`, then run that subset of files through `eslint --fix`, and then run that same subset through `git add`. In other words, unstaged files won't be affected.